addons/website_sale/views/templates.xml

```diff
@@ -53,6 +53,162 @@
+                                        <li><a href="#">Gadgets</a></li>
+                                    </t>
+                                </ul>
+                            </div>
+
+                            <!-- Column 2: Customer Care -->
+                            <div class="col-lg-2 o_colored_level mb-4 mb-lg-0">
+                                <h5>Customer Care</h5>
+                                <br/>
+                                <ul class="list-unstyled">
+                                    <li><a href="#">Help Center</a></li>
+                                    <li><a href="#">Returns</a></li>
+                                    <li><a href="#">Delivery</a></li>
+                                    <li><a href="#">Shipping</a></li>
+                                    <li><a href="/contactus">Contact us</a></li>
+                                </ul>
+                            </div>
+
+                            <!-- Column 3: Badges, Social, Payments -->
+                            <div class="col-12 col-lg-5 offset-lg-1 o_colored_level">
+                                <!-- Service Badges -->
+                                <span
+                                    class="s_badge badge text-bg-primary border border-dark o_animable"
+                                    data-name="Badge"
+                                >
+                                    <i class="fa fa-dollar fa-fw o_not-animable"/> Low Prices
+                                </span>
+                                <span
+                                    class="s_badge badge text-bg-primary border border-dark o_animable"
+                                    data-name="Badge"
+                                >
+                                    <i class="fa fa-shopping-basket fa-fw o_not-animable"/>
+                                    Easy Returns
+                                </span>
+                                <span
+                                    class="s_badge badge text-bg-primary border border-dark o_animable"
+                                    data-name="Badge"
+                                >
+                                    <i class="fa fa-truck fa-fw o_not-animable"/> Fast Shipping
+                                </span>
+
+                                <!-- Social Media -->
+                                <div class="row">
+                                    <div
+                                        class="pt32 pb32 s_social_media text-start o_not_editable"
+                                        data-snippet="s_social_media"
+                                        data-name="Social Media"
+                                        contenteditable="false"
+                                    >
+                                        <h5
+                                            class="s_social_media_title d-block"
+                                            contenteditable="true"
```

Comment:
You should not have the contenteditable attribute in the template... otherwise you allow the visitor to type their text :|